domain-ee/ee-ep-merge-app/integration/conftest.py

```diff
@@ -18,6 +18,11 @@ def pytest_collection_modifyitems(items):
     items.sort(key=item_order.get)
 
 
+@pytest.fixture(autouse=True, scope='session')
```

Comment:
This fixture isn't needed. This app won't have access to the DLQ.

Reply:
Same as above